### PR TITLE
[BFN]  Update configuration files

### PR DESCRIPTION
**Pull request by KostiantynYarovyiBf**
#### Why I did it
Move a couple switch specific options in the p4_devices section of the conf file
to a new section, switch_options, and parse them from the conf file directly.
#### How I did it
Update configuration files for initialization barefoot SDK
#### How to verify it

- Build Barefoot platform
- Run Newport CTs

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [X] 202111

### CODE REVIEW

**Comment by KostiantynYarovyiBf** [comment]
@lguohan, @prgeor,  @sujinmkang, @vdahiya12 please review

File: device/accton/x86_64-accton_wedge100bf_32qs-r0/montara/switch-tna-sai.conf [hunk 1]
```diff
@@ -1,23 +1,15 @@
...
-            "sds_fw_path": "share/tofino_sds_fw/avago/firmware"
+            "sds_fw_path": "share/tofino_sds_fw/avago/firmware",
+            "instance": 0
         }
     ],
     "p4_devices": [
         {
             "device-id": 0,
-            "agent0": "lib/platform/x86_64-accton_wedge100bf_32qs-r0/libpltfm_mgr.so",
+            "agent0": "lib/platform/x86_64-accton_wedge100bf_32x-r0/libpltfm_mgr.so",
```
Comment (akokhan) [inline device/accton/x86_64-accton_wedge100bf_32qs-r0/montara/switch-tna-sai.conf:12]: please fix

File: device/ingrasys/x86_64-ingrasys_s9180_32x-r0/INGRASYS-S9180-32X/switch-tna-sai.conf [hunk 2]
```diff
@@ -1,23 +1,15 @@
...
-            "sds_fw_path": "share/tofino_sds_fw/avago/firmware"
+            "sds_fw_path": "share/tofino_sds_fw/avago/firmware",
+            "instance": 0
         }
     ],
     "p4_devices": [
         {
             "device-id": 0,
-            "agent0": "lib/platform/x86_64-ingrasys_s9180_32x-r0/libpltfm_mgr.so",
+            "agent0": "lib/platform/x86_64-accton_wedge100bf_32x-r0/libpltfm_mgr.so",
```
Comment (akokhan) [inline device/ingrasys/x86_64-ingrasys_s9180_32x-r0/INGRASYS-S9180-32X/switch-tna-sai.conf:12]: please fix

File: device/wnc/x86_64-wnc_osw1800-r0/OSW1800-48x6q/switch-tna-sai.conf [hunk 3]
```diff
@@ -1,23 +1,15 @@
...
-            "sds_fw_path": "share/tofino_sds_fw/avago/firmware"
+            "sds_fw_path": "share/tofino_sds_fw/avago/firmware",
+            "instance": 0
         }
     ],
     "p4_devices": [
         {
             "device-id": 0,
-            "agent0": "lib/platform/x86_64-wnc_osw1800-r0/libpltfm_mgr.so",
+            "agent0": "lib/platform/x86_64-accton_wedge100bf_32x-r0/libpltfm_mgr.so",
```
Comment (akokhan) [inline device/wnc/x86_64-wnc_osw1800-r0/OSW1800-48x6q/switch-tna-sai.conf:12]: please fix

File: device/accton/x86_64-accton_wedge100bf_32qs-r0/montara/switch-tna-sai.conf (hunk 1 above)
Comment (KostiantynYarovyiBf) [reply device/accton/x86_64-accton_wedge100bf_32qs-r0/montara/switch-tna-sai.conf]: done

File: device/ingrasys/x86_64-ingrasys_s9180_32x-r0/INGRASYS-S9180-32X/switch-tna-sai.conf (hunk 2 above)
Comment (KostiantynYarovyiBf) [reply device/ingrasys/x86_64-ingrasys_s9180_32x-r0/INGRASYS-S9180-32X/switch-tna-sai.conf]: done

File: device/ingrasys/x86_64-ingrasys_s9280_64x-r0/INGRASYS-S9280-64X/switch-tna-sai.conf [hunk 4]
```diff
@@ -1,23 +1,15 @@
...
-            "sds_fw_path": "share/tofino_sds_fw/avago/firmware"
+            "sds_fw_path": "share/tofino_sds_fw/avago/firmware",
+            "instance": 0
         }
     ],
     "p4_devices": [
         {
             "device-id": 0,
-            "agent0": "lib/platform/x86_64-ingrasys_s9280_64x-r0/libpltfm_mgr.so",
+            "agent0": "lib/platform/x86_64-accton_wedge100bf_32x-r0/libpltfm_mgr.so",
```
[the remark this answers is not in the document]
Comment (KostiantynYarovyiBf) [reply device/ingrasys/x86_64-ingrasys_s9280_64x-r0/INGRASYS-S9280-64X/switch-tna-sai.conf]: done

File: device/wnc/x86_64-wnc_osw1800-r0/OSW1800-48x6q/switch-tna-sai.conf (hunk 3 above)
Comment (KostiantynYarovyiBf) [reply device/wnc/x86_64-wnc_osw1800-r0/OSW1800-48x6q/switch-tna-sai.conf]: done

**Review by akokhan — APPROVED** [review]

**Comment by KostiantynYarovyiBf** [comment]
@lguohan, @prgeor, @sujinmkang, @vdahiya12 please review and merge

**Comment by KostiantynYarovyiBf** [comment]
@lguohan, @prgeor, @sujinmkang, @vdahiya12 please review and merge

**Comment by KostiantynYarovyiBf** [comment]
@lguohan, @prgeor, @sujinmkang, @vdahiya12, @qiluo-msf , @xumia  please review and merge

**Review by msosyak — APPROVED** [review]
@qiluo-msft Please merge

**Comment by vboykox** [comment]
@lguohan please merge.

**Comment by vboykox** [comment]
@lguohan @qiluo-msft please merge.

**Pull request merged**